docs/platform/howto/disk-autoscaler.rst

```diff
@@ -0,0 +1,287 @@
+Scale your Aiven service disks automatically
+============================================
+
+Discover the service disk autoscaler and its capabilities. Find out how it works and how to use it with your Aiven services.
+
+.. topic:: Pricing
+
+   Costs of using disk autoscaler depend on your service type and plan. You're only charged based on actual storage usage on your service. For more details on the costs of using disk autoscaler, check the same for dynamic disk sizing (DDS) in `Aiven Plans and Pricing <https://aiven.io/pricing?product=kafka>`_ or contact us at sales@Aiven.io.
+
+Why use disk autoscaling
+------------------------
+
+Service disk autoscaler increases disk storage capacity automatically when the disk is running out of space. Disk autoscaling allows you to improve the cost-efficiency of operating your Aiven services: you can start with a relatively small-sized disk and only have it scaled up when needed with no risk of running out of disk space at any point.
+
+How it works
+------------
+
+There are a few steps illustrating how disk autoscaler works:
+
+1. You create an disk autoscaler integration endpoint in your Aiven project setting the maximum additional storage at the same time.
```

Comment:
```suggestion
1. You create a disk autoscaler integration endpoint in your Aiven project setting the maximum additional storage at the same time.
```

1. You create a disk autoscaler integration endpoint in your Aiven project setting the maximum additional storage at the same time.